### PR TITLE
 change url in docs

### PR DESCRIPTION
**Pull request by Ola2808-Boro**
#### Pull Request purpose

[ ] Documentation update
[X ] Bug fix
[ ] New rule
[ ] Updates an existing rule
[ ] Add something to the core
[ ] Other, please explain:

#### Checklist

[ ] Update CHANGELOG.md
[ ] Update task status in release plan (README)
[ ] Docs added (/docs/rules/<rule_name>)
[ ] Docs ready to be added to website (ec0lint.com)

### CODE REVIEW

File: lib/rules/avoid-lodash.js [hunk 1]
```diff
@@ -24,12 +24,13 @@ module.exports = {
             description: "disallow importing lodash",
             category: "Code improvements",
             recommended: false,
-            url: "https://eslint.org/docs/rules/avoid-lodash"
+            url: "https://www.ec0lint.com/features/avoid-lodash",
```
Comment (malwinq) [inline lib/rules/avoid-lodash.js]: it should use only http (without www)

Comment (malwinq) [reply lib/rules/avoid-lodash.js]: +all other rules

Comment (Ola2808-Boro) [reply lib/rules/avoid-lodash.js]: fix url for all other rules and add ec0lint's url to other rules which will have docs in the future

**Comment by malwinq** [comment]
thank you @Ola2808-Boro and welcome on board! all changes look good to me, OK to merge

**Review by malwinq — APPROVED** [review]

**Pull request merged**